core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceMultipartTest.java

```diff
@@ -78,6 +88,33 @@ void testUploadFile(String path) throws Exception {
                            "\"param1\":\"armeria\"}");
     }
 
+    @Test
+    void emptyBodyPart() {
+        final Multipart multipart = Multipart.of();
+        final HttpRequest request = multipart.toHttpRequest("/uploadWithMultipartObject");
```

Comment:
Should we add a unit test for an empty body to `MimeParserTest`?

Reply:
Thanks! added. 😉